doc/releases/changelog-0.36.0.md

```diff
@@ -68,13 +95,15 @@
         return qml.state()
     ```
 
-    As above, we can obtain the total spectral norm error of the circuit using `qml.specs`:
+    Again, the total spectral norm error of the circuit can be calculated using `qml.specs`:
 
     ```pycon
     >>> qml.specs(circuit)()["errors"]
     {'SpectralNormError': SpectralNormError(0.07616666666666666)}
     ```
 
+  Check out our amazing demo on using these new features in a real-world example!
```

Comment:
Need link

Reply:
It's been merged in but the public link won't be available until Tuesday